README.md

```
@@ -208,6 +602,263 @@
+   - Copy the username without @ symbol (e.g., `mycompanybot`)
+   - **BENEFIT**: Eliminates 200+ API calls, makes deep links instant
+
+3. **Backup Current Configuration**
+   ```bash
+   # Backup your current .env file
+   cp .env .env.backup
+   ```
+
+#### **🚀 Migration Steps**
+
+**Step 1: Update Environment Variables**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)